applications/luci-app-alist/htdocs/luci-static/resources/view/alist/config.js

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: Apache-2.0
+
+'use strict';
+'require form';
+'require poll';
+'require rpc';
+'require uci';
+'require validation';
+'require view';
+
+var callServiceList = rpc.declare({
+	object: 'service',
+	method: 'list',
+	params: ['name'],
+	expect: { '': {} }
+});
+
+function getServiceStatus() {
+	return L.resolveDefault(callServiceList('alist'), {}).then(function (res) {
+		var isRunning = false;
+		try {
+			isRunning = res['alist']['instances']['instance1']['running'];
+		} catch (e) { }
+		return isRunning;
+	});
+}
+
+function renderStatus(isRunning, port) {
+	var spanTemp = '<span style="color:%s"><strong>%s %s</strong></span>';
+	var renderHTML;
+	if (isRunning) {
+		var button = String.format('&#160;<a class="btn cbi-button" href="http://%s:%s" target="_blank" rel="noreferrer noopener">%s</a>',
+			window.location.hostname, port, _('Open Web Interface'));
+		renderHTML = spanTemp.format('green', _('AList'), _('RUNNING')) + button;
+	} else {
+		renderHTML = spanTemp.format('red', _('AList'), _('NOT RUNNING'));
+	}
+
+	return renderHTML;
+}
+
+var stubValidator = {
+	factory: validation,
+	apply: function(type, value, args) {
+		if (value != null)
+			this.value = value;
+
+		return validation.types[type].apply(this, args);
+	},
+	assert: function(condition) {
+		return !!condition;
+	}
+};
+
+return view.extend({
+	load: function() {
+		return Promise.all([
+			uci.load('alist')
+		]);
+	},
+
+	render: function(data) {
+		var m, s, o;
+		var webport = uci.get(data[0], 'config', 'listen_http_port') || '5244';
+
+		m = new form.Map('alist', _('AList'),
+			_('A file list/WebDAV program that supports multiple storages, powered by Gin and Solidjs.<br />' +
+				'Default login username is <code>admin</code> and password is <code>password</code>.'));
```

Comment:
Put the blocks in code as %s variables via a format() function.

Reply:
done